### ISSUE TITLE
Raid History Enhancements

### ISSUE THREAD

**Issue by RedBeardEth** [BibliothecaDAO/realms-react#212]
![image](https://user-images.githubusercontent.com/90423049/187123588-752ba81b-5cb7-4e95-820a-4fec3dbc6cf4.png)

Past raids are currently accessible from the realm details history panel and empire page, but an expanded individual raid view (with URL accessibility) should be implemented

- [ ] Add a string of sayings e.g "The Sacking of Shomlom" to name each raid (might need to assign this on Indexer or on-chain so names don't change each time viewed) 
- [ ]  Show initial army compositions, and units/vitality lost by each side
- [ ]  Links for social sharing

**Comment by ponderingdemocritus** [comment]
assigning to @RedBeardEth

**Comment by ponderingdemocritus** [comment]
closing this as stack has changed
